src/Native/System.Native/pal_errno.cpp

```diff
@@ -339,6 +347,7 @@ extern "C" int32_t SystemNative_ConvertErrorPalToPlatform(Error error)
         case PAL_EXDEV:
             return EXDEV;
         case PAL_ENONSTANDARD:
+        default:
```

Comment:
@jasonwilliams200OK, I think I'm not explaining myself well.

We explicitly did not want a default case in the switch statement.  As the comment below states, it was explicitly written this way such that if we missed any cases, the compiler would warn about it.  In other words, the warning you're trying to suppress by adding the default case is exactly the warning we want to happen by not having a default case.

Let's take a step back.  Can you explain the core problem again?  The PR states simply ```NOTRECOVERABLE and EOWNERDEAD are not available in NetBSD```.  Does that mean:
a) These named values don't exist but the system may still use those errno numbers for the same thing, or
b) These named values don't exist and the system will never use those values
?

Assuming it means (b), then the declaration of PAL_EOWNERDEAD and PAL_ENOTRECOVERABLE should also be ifdef'd out of the enum.  Then we're only including those PAL_ values on platforms that have the corresponding errno values, and there's no need for the default case or the extra compiler flags you added.

If it's not (b), can you help me to understand better why it is the way it is?